### DISCUSSION TITLE
components/base/BaseSearchBar

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#756] [Announcements]
# components/base/BaseSearchBar

基础搜索栏 常见的table表格上的查询搜索功能。可以按照不同的配置，实现不同的功能，如：搜索框默认值，搜索框类型，搜索框提示文字，搜索框默认值等。 Api BarProps 名称 类型 默认值 说明 必传 ------------------------------ config BaseFormItemType[] true 搜索条件 N conf...

http://192.168.5.19:8081/components/base/BaseSearchBar.html

**Comment by shenohe** [comment]
showshowway
